public/CHANGELOG.md

```diff
@@ -7,9 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Updated
```

Comment:
Updating two changelogs can be a pain, so add a note in the readme and the github pull request template so it's not missed if this is the approach moving forward.

Reply:
I wouldn't say this is an approach I'd like to continue moving forward. I accidentally ran "npm run build" while working on this and updated the file. I figured it wouldn't hurt to include the updates but think it's overkill to update it with _every_ pr.